types/templates.go

```diff
@@ -403,7 +401,6 @@ type ValidatorPageData struct {
 	Watchlist                                []*TaggedValidators
 	SubscriptionFlash                        []interface{}
 	User                                     *User
-	AverageAttestationInclusionDistance      float64
```

Comment:
**minor:** There is still an error message left that uses this variable name.
It is weirdly worded as "efficiency", "effectiveness" and "average inclusion distance" are kind of used for the same values. In any case I would rewrite the error message since AverageAttestationInclusionDistance is not something you try to retrieve that would cause this error.